DotNetRu.Android/DotNetRu.Android.csproj

```diff
@@ -109,8 +109,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="FormsViewGroup, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Xamarin.Forms.2.3.2.127\lib\MonoAndroid10\FormsViewGroup.dll</HintPath>
-      <Private>True</Private>
```

Comment:
Why did you remove <Private>True</Private>?
With that attribute, the assembly won't be copied to the output directory.

Reply:
I didn't do it manually. It's removed automatically when new version of XF was installed

Reply:
@bruce-willis OK, there might be a good reason for it. Let's keep it as it is for now